sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/src/Vendoring/OpenTelemetry.Instrumentation.AspNetCore/GrpcTagHelper.cs

```diff
@@ -0,0 +1,89 @@
+// <copyright file="GrpcTagHelper.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System.Diagnostics;
+using System.Text.RegularExpressions;
+using OpenTelemetry.Trace;
+
+namespace OpenTelemetry.Instrumentation.GrpcNetClient;
+
+internal static class GrpcTagHelper
```

Comment:
what is this class for? Seems like it's not from the libraries we're trying to vendor.

Reply:
Please check csproj of OpenTelemetry ASP.NET Core Intrumentation.

Reply:
We could consider removing this instrumentation from distro. Grpc server instrumentation will not be part of the stable instrumentation release.

Reply:
Don't want to do any customizations here. Just to keep the net6 target happy we copied over this code.